test/Evaluate/test_folding.sh

```diff
@@ -19,26 +19,21 @@
 #   - test_x is not folded (it is neither .true. nor .false.). This means the
 #     compiler could not fold the expression.
 
-PATH=/usr/bin:/bin
-srcdir=$(dirname $0)
-F18CC=${F18:-../../../tools/f18/bin/f18}
-CMD="$F18CC -fdebug-dump-symbols -fparse-only"
+CMD="$2 -fdebug-dump-symbols -fparse-only"
 
 if [[ $# < 1 ]]; then
```

Comment:
Same comment as common.sh. The minimum number of arguments has changed to 3, so we should print usage for less than 3.